lua/DCS-gRPC/grpc.lua

```diff
@@ -131,6 +131,7 @@ GRPC.errorPermissionDenied = function(msg)
   }
 end
 
+GRPC.event = grpc.event
```

Comment:
Could you add a comment here explaining why this line is present. I know you mentioned it in the summary but it would save a future dev a round-trip to the `blame` command and commit history.